safe/definitions/post_processors.py

```diff
@@ -292,11 +292,11 @@ def post_processor_displacement_function(
     'key': 'post_processor_gender',
     'name': tr('Gender Post Processor'),
     'description': tr(
-        'A post processor to calculate the number of affected females. '
+        'A post processor to calculate the number of displaced females. '
         '"Female" is defined as: ' + concepts['female']['description']
```

Comment:
It would be good to add a line here like:

'"Displaced" is defined as:  ' + concepts['displaced']